### PR TITLE
Optimize bg

### PR DESCRIPTION
**Pull request by T-Damer**
- Use css backgrounds instead of svg/webp

### CODE REVIEW

File: src/components/PrivacyWorld.tsx [hunk 1]
```diff
@@ -1,25 +1,61 @@
...
 const privacyWorld = classnames(
+  position('relative'),
   margin('md:my-60', 'my-32'),
   wordBreak('break-words'),
   maxWidth('max-w-full'),
   zIndex('z-10')
 )
 
 export default function () {
+  const { ref, inView } = useInView({ threshold: 0.4 })
```
Comment (T-Damer) [inline src/components/PrivacyWorld.tsx:21]: You don't need `inView`, it was used for aniamtion

File: src/components/Background.tsx [hunk 2]
```diff
@@ -1,33 +1,41 @@
...
+        position: 'absolute',
+        bottom: `${bottom}%`,
+        left: `${left}%`,
+        width: `${width}px`,
+        height: `${height}px`,
+        background: `radial-gradient(50% 50% at 50% 50%, ${background} 0%, ${background}00 100%)`,
+        rotate: `${rotate}deg`,
+        borderRadius: '100%',
+        filter: `blur(${blur}px)`,
+        opacity: reversedAppear ? (inView ? 0 : 0.6) : inView ? 0.6 : 0,
```
Comment (T-Damer) [inline src/components/Background.tsx:34]: You don't need `reversedAppear` prop at all, it was used for animation

File: src/components/Background.tsx [hunk 3]
```diff
@@ -1,33 +1,41 @@
...
+        transitionDuration: '4s',
```
Comment (T-Damer) [inline src/components/Background.tsx:35]: You don't need this, it's for animation

File: src/components/Background.tsx [hunk 4]
```diff
@@ -1,33 +1,41 @@
...
+        transitionProperty: 'opacity',
```
Comment (T-Damer) [inline src/components/Background.tsx:36]: You don't need this, it's for animation

File: src/components/LogoFooter.tsx [hunk 5]
```diff
@@ -8,13 +11,54 @@ import classnames, {
   flexDirection,
   justifyContent,
   margin,
+  position,
   space,
   zIndex,
 } from 'classnames/tailwind'
 
+const Backgrounds = ({ inView }: { inView: boolean }) => (
+  <BackgroundsAnimated inView={inView}>
```
Comment (smyzin) [inline src/components/LogoFooter.tsx:20]: Should we move all `Background`'s components to a separate dir? Like `components/Background/LogoFooter`, for instance

File: src/components/LearnBlock.tsx [hunk 6]
```diff
@@ -32,9 +37,63 @@ const dosuCityIcon = classnames(
...
+      />
+      <Background
+        width={2200}
+        height={650}
+        background="#CF00A3"
+        bottom={40}
+        left={-275}
+        rotate={-80}
+        blur={10}
+        saturate={2}
```
[the remark this answers is not in the document]
Comment (T-Damer) [reply src/components/LearnBlock.tsx:60]: SVG has a lot "magic" things inside too, it's just some positioning, sizing, based on designs

File: src/components/LogoFooter.tsx (hunk 5 above)
Comment (T-Damer) [reply src/components/LogoFooter.tsx]: Done :)

**Review by smyzin — APPROVED** [review]
Lgtm

**Pull request merged**